packages/renderer/MdToHtml/renderMedia.ts

```diff
@@ -39,6 +41,16 @@ export default function(link: Link, options: Options) {
 	}
 
 	if (options.pdfViewerEnabled && resource.mime === 'application/pdf') {
+		let anchorPageNo = null;
```

Comment:
this and the next few lines could probably go inside the `if (options.useCustomPdfViewer) {` condition as they aren't used outside of it